src/ui/training/trainingView.ts

```diff
@@ -154,20 +154,54 @@ function renderViewSolution(ctrl: TrainingCtrl) {
   }, i18n('viewTheSolution')) : h('div.buttonPlaceholder', '')
 }
 
+function renderVoteControls(ctrl: TrainingCtrl) {
+  if (ctrl.vm.voted === true) {
+    return [
+      h('span.fa.fa-caret-up'),
+      h('p', ctrl.getVotes()),
+      h('a', {
```

Comment:
I'd prefer a `button` instead of a `a`. It's not a website, and there is not much point to have an anchor tag without `href`.